docs/swarm/join-nodes.md

```diff
@@ -29,6 +29,11 @@ also run tasks.
 Before you add nodes to a swarm you must install Docker Engine 1.12 or later on
 the host machine.
 
+The Docker Engine joins the swarm according to the **join-token** you provide to
```

Comment:
> I assume there is a section on `Number of Managers in a swarm cluster`

See https://github.com/docker/docker/blob/master/docs/swarm/admin_guide.md#add-manager-nodes-for-fault-tolerance